### ISSUE TITLE
Exception: No softmax found.

### ISSUE THREAD

**Issue by Yung-zi** [OpenXAIProject/PyConKorea2019-Tutorials#2]
Have you had this problem?

When I tried to run this code

model = vgg16.VGG16()
model = innvestigate.utils.model_wo_softmax(model),

I met following error:
Exception: No softmax found.

I changed to this

model = innvestigate.utils.keras.graph.model_wo_softmax(model),

It doesn't account.

Thanks.
